integration/kotlinx-coroutines-play-services/README.md

```diff
@@ -0,0 +1,31 @@
+# Module kotlinx-coroutines-play-services
+
+Integration with Google Play Services [Tasks API](https://developers.google.com/android/guides/tasks).
+
+Extension functions:
+
+| **Name** | **Description**
+| -------- | ---------------
+| [Task.await][await] | Awaits for completion of the Task (cancellable)
+| [Deferred.asTask][asTask] | Converts a deferred value to a Task
+
+## Example
+
+Using Firebase APIs becomes simple:
+
+```kotlin
+scope.async {
```

Comment:
Why not use `launch` instead of `async` where the resulting `Deferred` is not used?
`async` should only be used if needed, and in the snippet here, it is not.

Reply:
Good point.